### PR TITLE
Desktop-version

### PR DESCRIPTION
**Pull request by Abenezer-Tilahun**
# Desktop version:- Created a Desktop version

# In this milestone;

- I applied media query to display the desktop view layout of the webpage
- I added a nav-bar with nav-items on the desktop view replacing the hamburger menu icon
- I adjusted the CSS required for the desktop view
-  Implemented Form-spree for form data collection

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,542 +1,767 @@
...
+  }
+
+  .item-navigation li a:hover::after {
+    width: 100%;
+  }
+
+  .header-text {
+    padding: 2rem 39rem;
+  }
+
```
Comment (elmar8287) [inline style.css]: - [ ] Please, consider the paddings and margins from the Figma template. Here I suggest you fix the padding to make your `header-text` class content closer to Figma design. You can see the difference between your design and Figma one on screenshots below:

> Your design
Screen Shot 2022-01-20 at 19 48 39

> Figma template
Screen Shot 2022-01-20 at 19 49 21

File: style.css [hunk 2]
```diff
@@ -1,542 +1,767 @@
...
   }
 
-  #name {
-    margin-left: 50px;
-    margin-top: 87px;
-    padding-left: 16px;
-    padding-right: 16px;
+  .contact-form {
+    padding-left: 51rem;
+    padding-right: 51rem;
```
Comment (elmar8287) [inline style.css]: - [ ] Here your form has an incorrect view in the desktop version. Look at the screenshot.
Screen Shot 2022-01-20 at 19 32 59

- To fix it, you can use `width: 447px;` as it mentioned in Figma template. I suggest deleting the paddings.

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @Abenezer-Tilahun ,

Great work on making the project requirements 👏 

There are still some requirements that aren't done yet, please work on it and ask for another review.
      
 ## Required Changes ♻️

- [ ] Please, consider my comments under review
- [ ] Please, consider, that your Pull Request is made from the “feature branch” into the main branch - it is the general requirement. In this case, you made the PR from `Desktop-version` to `Add-contact-form`

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
### Hi @Abenezer-Tilahun , 

## Status: Approved ✔️

:+1: You have implemented all the suggestions. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You may close this Pull Request and merge the changes**

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Amine Smahi_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -1,542 +1,766 @@
...
-@media only screen and (max-width: 480px) {
-  .contact-me {
+.contact-element:focus::placeholder {
+  color: #3a3c3f;
+}
+
+.btn-contact {
+  margin: 12px 90px;
+  background-color: #fff;
+}
```
Comment (medaminedev66) [inline style.css:531]: [OPTIONAL]
- I would recommend centering the submit button to improve the UI design.
image

File: style.css (hunk 1 above)
Comment (Abenezer-Tilahun) [reply style.css]: Ok I will adjust padding.

File: style.css (hunk 2 above)
Comment (Abenezer-Tilahun) [reply style.css]: OK i will do that thanks a lot for suggestion.

File: style.css (hunk 3 above)
Comment (Abenezer-Tilahun) [reply style.css:531]: Ok I will centering the submit button.

**Pull request merged**